reana-ui/src/pages/workflowDetails/components/WorkflowLogs.js

```diff
@@ -21,15 +21,26 @@ import { CodeSnippet, TooltipIfTruncated } from "~/components";
 
 import styles from "./WorkflowLogs.module.scss";
 
-export default function WorkflowLogs({ id }) {
-  const dispatch = useDispatch();
-  const loading = useSelector(loadingDetails);
-  const logs = useSelector(getWorkflowLogs(id));
-  const [selectedStep, setSelectedStep] = useState();
+function EngineLogs({ logs }) {
+  return logs ? (
+    <CodeSnippet dollarPrefix={false} classes={styles.logs}>
+      {logs}
+    </CodeSnippet>
+  ) : (
+    <Message
+      icon="info circle"
+      content="There are no workflow engine logs available."
```

Comment:
Minor: perhaps "There are no workflow engine logs available yet." message would better inform users that it will soon be populated. Or we add a sentence describing that the logs will be shown once the workflow finishes